src/sfdxCommand.ts

```diff
@@ -556,6 +563,28 @@ export abstract class SfdxCommand extends Command {
       });
     });
   }
+  private postToVarags(varargs: Dictionary<string | string[]>, name: string, value: string): void {
+    const descriptor = this.statics.varargs;
+    const multipleValuesAsArray = false || (!isBoolean(descriptor) && descriptor.multipleValuesAsArray);
+    const delimiter = !isBoolean(descriptor) && descriptor.delimiter ? descriptor.delimiter : ',';
+    if (!value) {
```

Comment:
Not sure if this will every be true. Should try to add a test for this. At a minimum this needs attention from the multiple keys perspective.